### PR TITLE
NONPR-3315: Upgrade nrs-retrieval to Play 3.0

### PR DESCRIPTION
**Pull request by AIbrahimQA**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,77 +1,66 @@
...
-  .settings(
-    Keys.fork in IntegrationTest := false,
-    Defaults.itSettings,
-    unmanagedSourceDirectories in IntegrationTest += baseDirectory(_ / "it").value,
-    parallelExecution in IntegrationTest := false
-  )
   .enablePlugins(PlayScala, SbtAutoBuildPlugin, SbtDistributablesPlugin)
   .disablePlugins(JUnitXmlReportPlugin)
+
+lazy val it = project
```
[the remark this answers is not in the document]
Comment (AIbrahimQA) [reply build.sbt:58]: Yeah, had to be separated out. Now you run the it tests like this `it/test`

[the remark this answers is not in the document]
Comment (DanielConnelly) [reply build.sbt:58]: so we can't simply run it:test?

[the remark this answers is not in the document]
Comment (AIbrahimQA) [reply build.sbt:58]: It doesn't recognise it:test anymore sadly

[the remark this answers is not in the document]
Comment (DanielConnelly) [reply build.sbt:58]: well thats sad, so we need a PR to update build-jobs aye?

[the remark this answers is not in the document]
Comment (DanielConnelly) [reply build.sbt:58]: I prefer it:test over it/test tbh :D but sbt knows best.

[the remark this answers is not in the document]
Comment (AIbrahimQA) [reply build.sbt:58]: we seem to only run test in the build jobs `.withTests('test')`. Integration tests should be added to this right?

**Review by DanielConnelly — DISMISSED** [review]

**Review by DanielConnelly — APPROVED** [review]

**Pull request merged**